embedding/cpp/elm_flutter_view.cc

```diff
@@ -0,0 +1,70 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "include/elm_flutter_view.h"
+
+#include "tizen_log.h"
+#include "utils.h"
+
+bool ElmFlutterView::RunEngine() {
+  if (!parent_) {
+    TizenLog::Error("The parent object is invalid.");
+    return false;
+  }
+
+  FlutterDesktopViewProperties view_prop = {};
+  view_prop.width = width_;
+  view_prop.height = height_;
+
+  // Read engine arguments passed from the tool.
+  Utils::ParseEngineArgs(&engine_args_);
+
+  std::vector<const char *> switches;
+  for (auto &arg : engine_args_) {
+    switches.push_back(arg.c_str());
+  }
+
+  FlutterDesktopEngineProperties engine_prop = {};
+  engine_prop.assets_path = "../res/flutter_assets";
+  engine_prop.icu_data_path = "../res/icudtl.dat";
+  engine_prop.aot_library_path = "../lib/libapp.so";
+  engine_prop.switches = switches.data();
+  engine_prop.switches_count = switches.size();
+
+  engine_ = FlutterDesktopEngineCreate(engine_prop);
+  if (!engine_) {
+    TizenLog::Error("Could not create a Flutter engine.");
+    return false;
+  }
+
+  view_ = FlutterDesktopViewCreateFromElmParent(view_prop, engine_, parent_);
```

Comment:
https://google.github.io/styleguide/cppguide.html#Local_Variables

> We encourage you to declare them in as local a scope as possible, and **as close to the first use as possible**.

The declaration of `view_prop` might be moved to here, similarly to `window_prop`'s case.

https://github.com/flutter-tizen/flutter-tizen/blob/61c1de7fc4c82c15d0964a79c6e9ea72f683f75b/embedding/cpp/flutter_app.cc#L43-L52

Reply:
Thanks. done.